ui/desktop/app/templates/application.hbs

```diff
@@ -1,17 +1,45 @@
 {{page-title (app-name)}}
 
-<Rose::Layout::Global as |layout|>
+<Rose::Layout::Global
+  class='{{if this.isWindowsOS ' layout-scroll'}}'
+  as |layout|
+>
 
   <layout.header>
-    <Rose::Header class='{{if (is-loading) ' loading'}}' as |header|>
+    <Rose::Header
+      class='{{if (is-loading) ' loading'}}
+        {{if this.isMacOS ' header-cushion'}}'
+      as |header|
+    >
       <header.brand @logo='logo-app' @text={{app-name}} />
-
       {{#if this.session.isAuthenticated}}
         <header.nav>
           {{outlet 'header-nav'}}
         </header.nav>
+      {{/if}}
 
-        <header.utilities as |utilities|>
+      <header.utilities as |utilities|>
+        {{#if this.isWindowsOS}}
+          <span class='utilities-actions'>
```

Comment:
This name feels too generic.  These are specifically window controls.  Is there a more specific name that makes sense?

Reply:
I was trying to keep in pattern with rose-like generic naming. Will reconsider.